### ISSUE TITLE
Summoned Spiritual Weapon uses strength as its modifier 

### ISSUE THREAD

**Issue by Nerdy-McHobbit** [kandashi/Macros#4]
Kandashi I'm a Patreon patron and I am grateful for all of the work that you do, with macros and in discord smoothing things out for everyone. Thank you for fielding questions.

The Summoned Spiritual Weapon created by the Spiritual Weapon Item Macro uses strength as it's ability modifier, not the default spellcasting ability if the ability modifier field is left at "default." 

In the images below, the only thing that differs is the strength has been bumped from 10 to 20. No ability mod bonus with str 10 and wis 22, ability mod present with str 20 wis 22.
![wis22str10](https://user-images.githubusercontent.com/83545166/116839527-4ff88d80-ab98-11eb-82a4-8aee16f036ec.png)
![wis22str20](https://user-images.githubusercontent.com/83545166/116839531-57b83200-ab98-11eb-8d15-f7d1fe922f94.png)

**Comment by kandashi** [comment]
Thanks for logging the error, I'll hopefully get a fix soon

**Comment by Nerdy-McHobbit** [comment]
Hm. And today I cast it and it has wisdom as the modifier.  I'm sorry, the problem must have been on my end.

**Comment by kandashi** [comment]
No, no you were correct. The DAE srd got a fix today for this issue and it should have come here too actually

**Comment by Nerdy-McHobbit** [comment]
Oh! Okay, great!

Would now be an okay time to point out that the template placed by the spell as the weapon won't carry the texture or effect selected in Token Magic FX, anymore? This was working before the fix.

**Comment by kandashi** [comment]
That is correct, the template is not placed by tmfx but by the macro, you can add your own texture by adding a line 
`texture: "path here",`
